src/index.js

```diff
@@ -100,47 +64,67 @@ var logToFile = function(input) {
                    });
 
   return fileName;
-}
+};
 
 var fileExists = function(filePath) {
   try {
     return fs.statSync(filePath).isFile();
   } catch (err) {
     return false;
   }
-}
+};
 
 // Callback for size and some input validity checks.
 var sizeCheckCallback = function(maxJobNumber, maxVehicleNumber) {
-  return function (req, res, next) {
-    var correctInput = ('jobs' in req.body)
-        && ('vehicles' in req.body);
-
+  return function(req, res, next) {
+    var correctInput = "jobs" in req.body && "vehicles" in req.body;
+    var message =
+      "Invalid JSON object in request, please add jobs and vehicles to the object body";
+    console.error(now() + ": " + JSON.stringify(message));
```

Comment:
This has to go within the following `if` to not get triggered for every request.